metagraph/src/annotation/representation/column_compressed/annotate_column_compressed.hpp

```diff
@@ -27,7 +27,9 @@ class ColumnCompressed : public MultiLabelEncoded<Label> {
     using VLabels = typename MultiLabelEncoded<Label>::VLabels;
 
     ColumnCompressed(uint64_t num_rows = 0,
-                     size_t num_columns_cached = 1);
+                     size_t num_columns_cached = 1,
+                     const std::string &swap_dir = "",
+                     uint64_t buffer_size = 1024 * 1024 * 1024 / sizeof(uint64_t));
```

Comment:
Even though it's clear from the default value that `buffer_size` is in GiB, maybe rename this to `buffer_size_gb` or modify the code to use bytes as units. The latter is more standard and could avoid confusion later.

Reply:
Good point, it's really unclear what the units are...

Reply:
I added a comment